### PR TITLE
Update to Bootstrap v5 - Additional Changes

### PR DESCRIPTION
**Pull request by dvvanessastoiber**
See https://github.com/datavisyn/visyn2025/issues/80

### CODE REVIEW

**Review by thinkh — APPROVED** [review]
Thanks for your work! The changes look good to me. 👍

File: src/scss/abstracts/_variables.scss [hunk 1]
```diff
@@ -23,6 +23,7 @@ $phovea-theme-colors: (
 $theme-colors: map-merge($theme-colors, $phovea-theme-colors);
 
 $phovea-loading-icon-url: url('~phovea_ui/dist/assets/caleydo_c_anim.svg') !default;
+$phovea-loading-icon-size: 10% !default;
```
Comment (thinkh) [inline src/scss/abstracts/_variables.scss:26]: The 10% seems to cause the a flickering in size when the view is loading. 

caleydo-logo-size

I guess that the percentage size is dependening on the size of a parent element and once the size changes the size of the background changes as well.

Maybe using `auto` (= browser default value) for the [`background-size`](https://developer.mozilla.org/en-US/docs/Web/CSS/background-size) is the better option here?

```suggestion
$phovea-loading-icon-size: auto !default;
```

Can you please have a look again?

**Review by thinkh — CHANGES_REQUESTED** [review]
Unfortunately, I still found a bug. Please see report below.

File: src/scss/abstracts/_variables.scss (hunk 1 above)
Comment (dvvanessastoiber) [reply src/scss/abstracts/_variables.scss]: I see you have already committed the fix. It works in my tests, thank you! :+1:

**Review by thinkh — APPROVED** [review]

**Pull request merged**